fabric/integrated_mcp_fabric.py

```
@@ -1,253 +1,246 @@
 
-                # Use mcp-use to create client
-                client = await create_client(
-                    server_url=url, client_name=f"{self.fabric_id}_{name}"
-                )
+            self._initialized = len(self.mcp_clients) > 0
+            return self._initialized
 
-                # Verify connection by listing tools
-                tools = await client.list_tools()
-                logger.info(f"Connected to {name} at {url} with {len(tools)} tools")
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)